src/crypto/CHIPCryptoPAL.h

```diff
@@ -28,9 +28,9 @@
 #endif
 
 #include <core/CHIPError.h>
-#include <support/CodeUtils.h>
 #include <stddef.h>
 #include <string.h>
+#include <support/CodeUtils.h>
```

Comment:
probably better to group support/ with core/ above

would look like:

```
#include <core/CHIPError.h>
#include <support/CodeUtils.h>
	
#include <stddef.h>
#include <string.h>
```

newline separating the groups is significant :-(